packages/webapp/src/components/Crop/PlantingLocation/index.jsx

```diff
@@ -196,6 +215,15 @@ export default function PurePlantingLocation({
             onChange={defaultLocationCheckboxOnChange}
           />
         )}
+        {showOrganicStausMismatchModal && (
+          <OrganicStatusMismatchModal
+            modalContent={modalContent}
+            dismissModal={(dismissStatus) => {
```

Comment:
In the case where dismissModal(0) ("Thats Fine") , the form should submit instead of returning to the "Set your location" view.